docs/one-click-integrations/bitbucket-integration.md

```diff
@@ -86,6 +112,17 @@ The table below lists the data that the Bitbucket integration collects from your
     </td>
     <td>Deployment frequency and Change failure rate on the <a href="../../metrics/accelerate/">Accelerate Overview dashboard</a></td>
 </tr>
+<tr>
```

Comment:
@afsmeira I added this new row for Incidents to the Collected data table. Can you please validate and complete it?

Reply:
incident_id: pull request id
timestamp_created: reverted pull request merged date
timestamp_resolved: pull request merged date
system: repository name

Reply:
Updated! 👍

Reply:
Almost correct. In Bitbucket, PRs don't have unique IDs (like in GitHub) so we use the PR number. We also add some timestamp information to the incident ID but I don't think it's relevant to mention here.

Note that the incident id uses the number of the current pull request, not of the reverted pull request.

Incident ids are only relevant for the 'Events Log' page, they don't affect metrics.